proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol

```diff
@@ -288,12 +510,137 @@ contract DepositBoxERC20 is DepositBox, IDepositBoxERC20 {
         returns (address[] memory tokensInRange)
     {
         require(
-            from < to && to - from <= 10 && to <= _schainToERC20[keccak256(abi.encodePacked(schainName))].length(),
+            from < to && to - from <= 10 && to <= _schainToERC20[_schainHash(schainName)].length(),
             "Range is incorrect"
         );
         tokensInRange = new address[](to - from);
         for (uint256 i = from; i < to; i++) {
-            tokensInRange[i - from] = _schainToERC20[keccak256(abi.encodePacked(schainName))].at(i);
+            tokensInRange[i - from] = _schainToERC20[_schainHash(schainName)].at(i);
+        }
+    }
+
+    /**
+     * @dev Get amount of tokens that are delayed for specified receiver
+     */
+    function getDelayedAmount(address receiver, address token) external view override returns (uint256 value) {
+        uint256 delayedTransfersAmount = delayedTransfersByReceiver[receiver].length();
+        for (uint256 i = 0; i < delayedTransfersAmount; ++i) {
+            DelayedTransfer storage transfer = delayedTransfers[uint256(delayedTransfersByReceiver[receiver].at(i))];
```

Comment:
Why "storage"?

Reply:
Not all fields are needed in memory. So it's cheaper.